src/components/elements/typography/index.js

```diff
@@ -0,0 +1,115 @@
+import { Link as GatsbyLink } from 'gatsby'
+import styled from 'styled-components'
+import { letterSpacing, colors, size, psize, lineHeight } from '../../../utils'
+
+const { blueCharcoal, mineShaft } = colors
+
+export const H1 = styled.h1`
+  font-size: ${size.xl};
+  letter-spacing: ${letterSpacing.medium};
+  white-space: nowrap;
+  font-weight: 800;
+`
+
+export const H2 = styled.h2`
+  font-size: ${size.lg};
+  letter-spacing: ${letterSpacing.medium};
+  font-weight: 700;
+`
+
+export const H3 = styled.h3`
+  font-size: ${size.md};
+  letter-spacing: ${letterSpacing.small};
+  font-weight: 700;
+`
+
+export const H4 = styled.h4`
+  font-size: ${size.xs};
+  letter-spacing: ${letterSpacing.small};
+  font-weight: 700;
+`
+
+export const CapsHeadingLarge = styled.h1`
+  font-size: ${size.s};
+  letter-spacing: ${letterSpacing.huge};
+  white-space: nowrap;
+  font-weight: ${({ w = 700 }) => w};
+  text-transform: uppercase;
+`
+
+export const CapsHeadingMedium = styled.h2`
+  font-size: ${size.xs};
+  letter-spacing: ${letterSpacing.medium};
+  white-space: nowrap;
+  text-transform: uppercase;
+  color: ${mineShaft};
+`
+
+export const CapsHeadingSmall = styled.h3`
+  font-size: ${size.xxs};
+  letter-spacing: ${({ l = letterSpacing.medium }) => l};
+  white-space: nowrap;
+  font-weight: 500;
+  text-transform: uppercase;
+`
+
+// Paragraphs
+// naming: {Bold}?{Size}P, examples: XLP: x-large paragraph
+
+export const XLP = styled.p`
+  font-size: ${psize.xl};
+  line-height: ${lineHeight.xl};
+  font-weight: 400;
+`
+
+export const LP = styled.p`
+  font-size: ${psize.l};
+  line-height: ${lineHeight.l};
+  font-weight: 400;
+`
+
+export const MP = styled.p`
+  font-size: ${psize.m};
+  line-height: ${lineHeight.m};
+  font-weight: 400;
+`
+
+export const BoldMP = styled.p`
+  font-size: ${psize.m};
+  line-height: ${lineHeight.l};
+  font-weight: 700;
+`
+
+export const SP = styled.p`
+  font-size: ${psize.s};
+  line-height: ${lineHeight.s};
+  font-weight: 300;
+`
+
+export const BoldXSP = styled.p`
+  font-size: ${psize.xs};
+  line-height: 1.8rem;
+  font-weight: 700;
+`
+
+export const XSP = styled.p`
+  font-size: ${psize.xs};
+  line-height: 1.8rem;
+  font-weight: 300;
+`
+
+export const P = styled.p`
+  font-size: ${({ s = size.s }) => s};
+  font-weight: 400;
+`
+
+export const Link = styled(GatsbyLink)`
```

Comment:
Just noticed this, haven't got any links in the design currently.

Reply:
Yes, but we might use them because it was already on the first page so I styled it up. We can remove it if that's unnecessary

Reply:
Okay cool, will design this when it's necessary for the text and then can change here - will most probably need one in the earwig page